log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/Date_Range_Picker/js/main.js

```diff
@@ -0,0 +1,142 @@
+var href = parent.window.location.href,
+    hrefLastSegment = href.substr(href.lastIndexOf('/') + 1),
+    resolveURI = parent.ues.global.dashboard.id == hrefLastSegment ? '../' : '../../';
+
+var TOPIC = "range-selected";
+$(function() {
+    var dateLabel = $('#reportrange .btn-label');
+    //if there are url elemements present, use them. Otherwis use last hour
+    var timeFrom = moment().subtract(29, 'days');
+    var timeTo = moment();
+    var message = {};
+
+    var qs = gadgetUtil.getQueryString();
+    if (qs.timeFrom != null) {
+        timeFrom = qs.timeFrom;
+    }
+    if (qs.timeTo != null) {
+        timeTo = qs.timeTo;
+    }
+    var count = 0;
+    console.log("TimeFrom: " + timeFrom + " TimeTo: " + timeTo);
```

Comment:
Hi Sinthuja,

This was fixed by the commits in https://github.com/ksdperera/shared-analytics/commit/0ebea7478d812bde0952e207fcd99f08003ca3f2#diff-be0f0a08edcae8c976c8766b92f8a79cL23

Thanks,
SajithD